source/help/settings/custom-emoji.rst

```diff
@@ -2,19 +2,37 @@
 Custom Emoji
 ------------------
 
-**Note:** If you cannot see the **Custom Emoji** option in the menu, then your System Administrator may have restricted access to certain users. Contact your System Administrator for help.
+You can create Custom Emojis which are available to everyone on your Mattermost server. 
 
-Settings for Custom Emoji are accessible from the **Main Menu** by clicking the hamburger menu at the top of the channels sidebar. Clicking **Custom Emoji** opens a page where you can view existing Custom Emojis and create new ones.
+To create and find Custom Emojis, open the **Main Menu** at the top right of the channels sidebar and select **Custom Emoji**.
 
-To create a Custom Emoji:
+**Note:** If you cannot see the **Custom Emoji** option in the menu, then your Mattermost System Admin may have restricted access to certain users. Contact your Mattermost System Admin for help.
 
-1. Click on the hamburger menu at the top of the channels sidebar to open the **Main Menu**.
+Creating Custom Emojis
+^^^^^^^^^^^^^^^^^^^^^^
+
+Small, square pictures work best when selecting an image. The file can be any JPG, GIF, or PNG and up to 1 MB in size.
+
+1. Open the **Main Menu** at the top-right of the channels sidebar.
 2. Select **Custom Emoji**.
-3. Click **Add Custom Emoji**.
-4. Enter a name for your Custom Emoji. (This is the name that will show up in the emoji autocomplete).
-5. Click **Select**, and choose what image to use for the emoji.
-6. Click **Save**.
+3. Choose **Add Custom Emoji**.
+4. Enter a name for your Custom Emoji. This is the name that will show up in the emoji autocomplete.
+5. Choose **Select**, and choose what image to use for the emoji. 
+6. Choose **Save**.
+
+Once saved, your emoji will be added to the list of Custom Emoji. 
+
+To use your Custom Emoji in a message, simply type ":" followed by your emoji name to bring it up in the emoji autocomplete. Other users on the site will also be able to see your emoji in the autocomplete, and will be able to use it in their own messages.
 
-Small, square pictures work best when selecting an image. The file can be any JPG, GIF, or PNG.
+Deleting Custom Emojis
+^^^^^^^^^^^^^^^^^^^^^^^
 
-After saving, your emoji will show up in the list of Custom Emoji. To add your Custom Emoji to a message, simply type ":" followed by your emoji name to bring it up in the emoji autocomplete. Other users on the site will also be able to see your emoji in the autocomplete, and will be able to use it in their own messages.
+You can delete Custom Emojis that you created. 
+
+.. image:: ../images/delete_custom_emoji.png
```

Comment:
```suggestion
.. image:: ../../images/delete_custom_emoji.png
```

@justinegeffen You'll need to add a second `../` so that it gets the image from the correct folder relative to where the `custom-emoji.rst` page is located in the repo

Reply:
Related, I don't mean to create extra work for this PR, but it'll be good to review the screenshot follows our screenshot guidelines here: https://handbook.mattermost.com/operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs

Apologies as I don't think I shared these guidelines before.

Reply:
Ah! Thank you!! That's extremely helpful and thanks for fixing the image link. 

It's not too much extra work - I'll get that set up now and redo it. :D